### PR TITLE
Refactor/switch

### PR DESCRIPTION
**Pull request by dmscn**
Updating `Checkbox.Switch` according to Figma protype [updates](https://www.figma.com/file/wn9grDi9mM29Wt7LmQjol7/Yoga-Design-System-2.0?node-id=4670%3A7034).

### CODE REVIEW

File: packages/yoga/src/Checkbox/CheckboxSwitch.theme.js [hunk 1]
```diff
@@ -36,15 +36,31 @@ const CheckboxSwitch = ({
...
-      backgroundColor: hexToRgb(colors.elements.selectionAndIcons, 0.2),
+      backgroundColor: hexToRgb(colors.elements.lineAndBorders, 0.25),
+    },
+  },
+  hover: {
+    checked: {
+      backgroundColor: hexToRgb(colors.text.secondary, 0.25),
+    },
+    disabled: {
+      backgroundColor: hexToRgb(colors.elements.lineAndBorders, 0.25),
```
Comment (allyssonsantos) [inline packages/yoga/src/Checkbox/CheckboxSwitch.theme.js:55]: According to figma, this color should be `colors.elements.backgroundAndDisabled`
```suggestion
      backgroundColor: hexToRgb(colors.elements.backgroundAndDisabled, 0.25),
```

File: packages/yoga/src/Checkbox/CheckboxSwitch.theme.js [hunk 2]
```diff
@@ -36,15 +36,31 @@ const CheckboxSwitch = ({
...
     },
   },
   focus: {
     checked: {
-      backgroundColor: hexToRgb(colors.primary, 0.2),
+      backgroundColor: hexToRgb(colors.text.secondary, 0.5),
     },
     disabled: {
-      backgroundColor: hexToRgb(colors.elements.selectionAndIcons, 0.2),
+      backgroundColor: hexToRgb(colors.elements.lineAndBorders, 0.25),
```
Comment (allyssonsantos) [inline packages/yoga/src/Checkbox/CheckboxSwitch.theme.js:47]: According to figma, this color should be `colors.elements.backgroundAndDisabled` and the opacity should be `0.5`
```suggestion
      backgroundColor: hexToRgb(colors.elements.backgroundAndDisabled, 0.5),
```

**Review by allyssonsantos — CHANGES_REQUESTED** [review]

File: packages/yoga/src/Checkbox/CheckboxSwitch.theme.js (hunk 1 above)
Comment (dmscn) [reply packages/yoga/src/Checkbox/CheckboxSwitch.theme.js:55]: You're right!

**Review by ggdaltoso — CHANGES_REQUESTED** [review]
We should revisit Switch on native cause there is no _thumb_ when `disabled`

image

**Review by ggdaltoso — APPROVED** [review]

**Review by caioalexandrebr — APPROVED** [review]

**Review by allyssonsantos — APPROVED** [review]

**Pull request merged**